### DISCUSSION TITLE
Nevertheless input disabled, yup resolver check validation. is it right? I don't understand.

### DISCUSSION THREAD

**Discussion by kyujonglee** [react-hook-form/react-hook-form#3742] [Q&A]
This is same as document example.
Some difference is that first name input is disabled.

https://codesandbox.io/s/react-hook-form-v6-validationschema-forked-6d9fh?file=/src/index.js

But, yup resolver still check first name input.

Is this the right?

**Comment by bluebill1049** [comment]
I think you are after `readOnly` instead of `disabled`.
